app/api/helpers/storage.py

```diff
@@ -162,7 +162,9 @@ def upload_local(uploaded_file, key, **kwargs):
         return get_settings()['static_domain'] + \
             file_relative_path.replace('/static', '')
     url = urlparse(request.url)
-    return url.scheme + '://' + url.hostname + file_relative_path
+    return '{scheme}://{hostname}:{port}{file_relative_path}'.format(
+        scheme=url.scheme, hostname=url.hostname, port=url.port,
+        file_relative_path=file_relative_path).replace(':None', '')
```

Comment:
can we replace `:None` here with the empty string? This would remove the need for a generic placeholder that can be stripped

Reply:
@maxlorenz So `:None` should be retained?

Reply:
Well `:None` is not a valid port while the empty string is so I guess having `:None` does not really do much here?